docs/user/dql/functions.rst

```diff
@@ -2037,6 +2037,56 @@ Example::
     | 13:49:00           | 13:49:00        | 13:49:00                                 | 13:49:00                      |
     +--------------------+-----------------+------------------------------------------+-------------------------------+
 
+TIME_FORMAT
+-----------
+
+Description
+>>>>>>>>>>>
+
+Usage: time_format(time, format) formats the time argument using the specifiers in the format argument.
+
```

Comment:
Add limitations there and describe how `DATE` processed

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/204/commits/8a2279a9734e063e7105fb7915f0fa0fb732a936